DATA_FORMAT.md

```diff
@@ -0,0 +1,372 @@
+        ...
+    ],
+    "min": <min value for all features>,
+    "max": <max value for all features>
+}
+```
+
+### 5. Centroids (optional)
+
+The centroids file defines the center of each object ID in the dataset. It follows the same format as the feature file, but each ID has two entries corresponding to the `x` and `y` coordinates of the object's centroid, making the `data` array twice as long.
+
+For each index `i`, the coordinates are `(x: data[2i], y: data[2i + 1])`.
+Coordinates are defined in pixels in the frame, where the upper left corner of the frame is (0, 0).
+
+```
+--centroids.json--
+{
+    "data": [
+        <x coordinate for id 0>,
+        <y coordinate for id 0>,
+        <x coordinate for id 1>,
+        <y coordinate for id 1>,
+        ...
+    ]
+}
+```
+
+### 6. Bounds (optional)
+
+The bounds file defines the rectangular boundary occupied by each object ID. Like centroids and features, the file defines a `data` array, but has four entries for each object ID to represent the `x` and `y` coordinates of the upper left and lower right corners of the bounding box.
+
+For each object ID `i`, the minimum bounding box coordinates (upper left corner) are given by `(x: data[4i], y: data[4i + 1])`, and the maximum bounding box coordinates (lower right corner) are given by `(x: data[4i + 2], y: data[4i + 3])`.
+
+Again, coordinates are defined in pixels in the image frame, where the upper left corner is (0, 0).
+
+```
+--bounds.json--
+{
+    "data": [
+        <upper left x for id 0>,
+        <upper left y for id 0>,
+        <lower right x for id 0>,
+        <lower right y for id 0>,
+        <upper left x for id 1>,
+        <upper left y for id 1>,
+        ...
+    ]
+}
+```
+
+### 7. Outliers (optional)
+
+The outliers file stores whether a given object ID should be marked as an outlier using an array of booleans (`true`/`false`). Indices that are `true` indicate outlier values, and are given a unique color in Nucmorph-Colorizer.
+
+```
+--outliers.json--
+{
+    "data": [
+        <whether track id 0 is an outlier>,
+        <whether track id 1 is an outlier>,
+        <whether track id 2 is an outlier>,
+        ...
+    ]
+}
+```
+
+<details>
+<summary><b>[Show me an example!]</b></summary>
+
+---
+
+For example, if a dataset had the following tracks and outliers, the file might look something like this.
+
+| Track # | Object IDs | Outliers |
+| ------- | ---------- | -------- |
+| 1       | 0, 1, 4    | 1        |
+| 2       | 2, 3, 5    | 2, 5     |
+
+```
+--outliers.json--
+{
+    "data": [
+        false, // 0
+        true,  // 1
+        true,  // 2
+        false, // 3
+        false, // 4
+        true   // 5
+    ]
+}
+```
+
+---
+
+</details>
+
+## Collections
+
+Collections are defined by an optional JSON file and group one or more datasets for easy access. Nucmorph-Colorizer can parse collection files and present its datasets for easier comparison and analysis from the UI.
```

Comment:
Nit/future task: we really need to rename this thing as it's not only for nucmorph anymore 😀